modules/search/instant-search/components/search-result-product.jsx

```diff
@@ -0,0 +1,64 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, Component } from 'preact';
+
+/**
+ * Internal dependencies
+ */
+import SearchResultComments from './search-result-comments';
+import PhotonImage from './photon-image';
+
+class SearchResultProduct extends Component {
+	render() {
+		const { result_type, fields, highlight } = this.props.result;
+		if ( result_type !== 'post' ) {
+			return null;
+		}
+
+		const firstImage = Array.isArray( fields[ 'image.url.raw' ] )
+			? fields[ 'image.url.raw' ][ 0 ]
+			: fields[ 'image.url.raw' ];
+
+		return (
+			<li className="jetpack-instant-search__search-result-product">
+				<h3>
+					<a
+						className="jetpack-instant-search__result-product-title"
+						href={ `//${ fields[ 'permalink.url.raw' ] }` }
+						onClick={ this.props.onClick }
+						rel="noopener noreferrer"
+						target="_blank"
+						//eslint-disable-next-line react/no-danger
+						dangerouslySetInnerHTML={ { __html: highlight.title } }
```

Comment:
Can you let me know where highlight.title is escaped ? E.g. endpoint that is returning the data for review.

Reply:
cc @gibrown

Reply:
Done in the endpoint with wp_kses()